src/components/Feed.jsx

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+
+const Feed = ({ tweets }) => {
+  return (
+    <div>
+      <h3>Post Feed</h3>
+      <ol>
+        {tweets.map((tweet) => (
+          <li key={tweet.id}>
+            <span className="userName">{tweet.userName}:</span> {tweet.text}
+            <br />
```

Comment:
Ok, maybe this came with the template code you were using. I guess it's more a preference thing, but in my opinion `br` tags can be avoided with good CSS. Because this can become something called an "antipattern" - basically a bad code habit. This can result in overusing it in too many places, cluttering the code. In addition, what is the difference between span and div? If you need to put the tweet date on the next line down, you should be using divs for userName and date, not spans.

Reply:
No, the `br` tag here was from me. I wanted a not so complicated way to put the date on a different line than tweet.text. I originally used `span` for the date because I wasn't sure if I wanted it to be on the same line as tweet.text or not. For some reason I didn't think of just switching to `div` here as I was playing around with the CSS.